tests/conftest.py

```diff
@@ -133,7 +174,7 @@ def user_token_fixture(user_cognito: dict) -> str:
     return encode_token(user_cognito)
 
 
-@pytest.fixture(scope="session", name="user")
+@pytest.fixture(name="user")  # scope="session", - TODO: bring that back to reduce run time
```

Comment:
Will we be able to bring that back if the `User` depends on configuration? I think a different local fixture will be required anyway to satisfy the problematic tests.